myapp/spec/system/tasks_spec.rb

```diff
@@ -0,0 +1,243 @@
+        expect(page).to have_field 'Status', with: 'TODO'
+        expect(page).to have_field 'Priority', with: 'Low'
+      end
+
+      context 'when user click "Create Task" button' do
+        it 'go to Task detail page' do # rubocop:disable RSpec/MultipleExpectations
+          click_on 'Create Task'
+          expect(page).to have_content '散歩'
+          expect(page).to have_content '多摩川を歩く'
+          expect(page).to have_content '2022-06-20'
+          expect(page).to have_content 'TODO'
+          expect(page).to have_content 'Low'
+        end
+      end
+    end
+  end
+
+  describe '#update' do
+    let(:task) { create(:task) }
+
+    before do
+      visit edit_task_path(task)
+    end
+
+    context 'when user edit task form correctly' do
+      before do
+        fill_in 'Task title', with: '運動'
+        fill_in 'Description', with: '多摩川を走る'
+        fill_in 'Limit', with: '2022-06-20'
+        select 'IN PROGRESS', from: 'Status'
+        select 'Normal', from: 'Priority'
+      end
+
+      it 'display new task info' do # rubocop:disable RSpec/MultipleExpectations
+        expect(page).to have_field 'Task title', with: '運動'
+        expect(page).to have_field 'Description', with: '多摩川を走る'
+        expect(page).to have_field 'Limit', with: '2022-06-20'
+        expect(page).to have_field 'Status', with: 'IN_PROGRESS'
+        expect(page).to have_field 'Priority', with: 'Normal'
+      end
+
+      context 'when user click "Update Task" button' do
+        it 'go to Task detail page' do # rubocop:disable RSpec/MultipleExpectations
+          click_on 'Update Task'
+          expect(page).to have_content '運動'
+          expect(page).to have_content '多摩川を走る'
+          expect(page).to have_content '2022-06-20'
+          expect(page).to have_content 'IN_PROGRESS'
+          expect(page).to have_content 'Normal'
+          expect(page).to have_content 'Edit success'
+        end
+      end
+    end
+
+    context 'when user edit task form incorrectly' do
+      before do
+        fill_in 'Task title', with: ''
+        fill_in 'Description', with: '多摩川を走る'
+        fill_in 'Limit', with: '2022-06-20'
+        select 'IN PROGRESS', from: 'Status'
+        select 'Normal', from: 'Priority'
+      end
+
+      it 'unable to complete editing (because name is blank)' do
+        click_on 'Update Task'
+        expect(page).to have_content 'Edit failed'
+      end
+    end
+  end
+
+  describe '#destroy' do
+    before do
+      create(:task)
+      visit root_path
+    end
+
+    context 'when the deletion process success' do
+      it 'delete task' do  # rubocop:disable RSpec/MultipleExpectations
+        expect {
+          click_on 'Delete'
+        }.to change { Task.count }.by(-1)
+      end
+
+      it 'display delete success message' do
+        click_on 'Delete'
+        expect(page).to have_content 'delete success'
+      end
+    end
+
+    context 'when the deletion process fails ' do
```

Comment:
fablicop(Rubocop)回したら指摘でましたが余計な空白が入ってしまってますね 👀 

```suggestion
    context 'when the deletion process fails' do
```

Reply:
rubocopのvscode設定にお時間いただきありがとうございます！
修正しました！
https://github.com/Fablic/training/pull/1330/commits/80cdd67e72f22d7e13cdb4e9f2bf8d668deea26a